### ISSUE TITLE
[Bug?]: Static Asset Imports Do Not Work With SSR

### ISSUE THREAD

**Issue by dairyisscary** [solidjs/solid-start#1182]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Current behavior 😯

Hi Ryan, Nikhil, and everyone else who's been putting in the amazing work into solid start (and the web at large haha)! 👋 

When importing static assets, for example, `import Img from "~/test.png";`, the default export of this appears to be incorrect when server rendering this string, resulting in a 404 for the browser.

Example server output:
```html
<!--!$e0-0-0-0-0-0--><img data-hk="0-0-0-0-0-0-0" src="/src/test.png" alt="hello world"><!--!$/e0-0-0-0-0-0-->
```

Client side rendering (with `{ start: { ssr: false } }`) appears to get this `src` attribute correct with a `/_build/@fs` prefixed url that does not 404. I didn't try client side navigation _within_ an SSRed app, but my suspicion is that it works since the client bundles are probably working the same in that case.

Also note that using hand-crafted, "public" urls from the vite `/public` directory appears to be work in both modes, unsurprisingly. 

### Expected behavior 🤔

Both server and client bundles should have same default export strings.

### Steps to reproduce 🕹

Minimal reproduction (this is my `app.jsx` file in a "bare" start setup) and `{ start: { ssr: true } }` in `vite.config.js`

```js
// @refresh reload
import img from "~/test.png";

export default function App() {
  return <img src={img} alt="hello world" />;
}
```

### Your environment 🌎

```shell
"@solidjs/start": "^0.4.2",
"solid-js": "^1.8.7",
"vinxi": "0.0.54"
```

**Comment by Christopher2K** [comment]
Experiencing the same thing using latest SolidStart!

**Comment by nksaraf** [comment]
Thanks for the report! Will take a look. Probably a bug in vinxi

**Comment by nksaraf** [comment]
Should be fixed in vinxi@0.0.55. please try that and reopen this issue if it still doesn't work.
